cluster/cluster_test.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 func TestPeersForQuery(t *testing.T) {
+	Mode = ModeMulti
```

Comment:
a few lines below this it says `Convey("when cluster in single mode", t, func() {`
whereas here we're testing multi mode with 1 node.
maybe we should make a separate test case for an actual single mode